src/ansys/aedt/core/filtersolutions_core/ideal_response.py

```diff
@@ -341,9 +341,9 @@ def frequency_response(
         y_axis_parameter: `FrequencyResponseColumn`, optional
              Frequency response column to return. The default is the frequency response magnitude in dB.
         minimum_frequency: str, optional
-            The default is ``200 MHz``.
+            The default is ``None``.
```

Comment:
We should describe the behavior of ``None``

Reply:
@ramin4667 I meant that we should describe what happens when the value is ``None``. It doesn't seem like this is obvious/unambiguous.

Reply:
Done

Reply:
@ramin4667 The updated comment mentions that the orignal default (e.g. 200 MHz) will be used if the value is None. But is that only true if the limit has never been set or is that true in general? For example, if someone sets the limit to 100MHz, and then calls the function and leaves the limit as None, will it get set back to 200MHz or stay at the 100MHz value? If it is set back to 200MHz, then I think your comment is correct, but if the limit stays at whatever value is currently set, then I think the comment needs to be corrected to make that clear.

Reply:
Updated.